### PR TITLE
Test hosts/nodes users and groups

### PR DESCRIPTION
**Pull request by rrasouli**
Purpose or Intent
=================
Testing the security section on infra provider nodes that currently includes users and groups

### CODE REVIEW

**Comment by rrasouli** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Lint report for commit bf417c9cd977651958d92fc5492f49c38e201c72:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Response to comment #249517273:

It seems like all the basic requirements were met, I have therefore merged your PR.

File: cfme/tests/openstack/test_host_security.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'],
+                                         scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_security(provider, soft_assert):
```
Comment (mfalesni) [inline cfme/tests/openstack/test_host_security.py]: This test does not do what it is supposed to do by its name. It says it checks *security* but in fact it just checks if SSA reveals any user and any group, so sorry but this is wrong.

Comment (rrasouli) [inline cfme/tests/openstack/test_host_security.py]: @mfalesni I can understand the name is confusing, however the test purpose to check that users and groups appear under the security section, indeed after SSA

Comment (mfalesni) [inline cfme/tests/openstack/test_host_security.py]: Wouldn't it make more sense to unify such tests in a test file called eg. test_host_summary where you would check various UI values?

**Pull request merged**